dynawo/sources/Models/Modelica/Dynawo/NonElectrical/Blocks/Continuous/DiscretePI.mo

```diff
@@ -19,24 +19,32 @@ block DiscretePI "Proportional integrator with discrete input"
 
   parameter Real Gain "Control gain";
   parameter Types.Time tIntegral "Time integration constant";
+  parameter Real Y0 "Start value of the PI output ";
 
-  discrete Interfaces.RealInput u "Input connector" annotation(
-    Placement(visible = true, transformation(origin = {-110, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
-
+  discrete Modelica.Blocks.Interfaces.RealInput u "Input connector" annotation(
+    Placement(visible = true, transformation(origin = {-110, -40}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(extent = {{-10, -10}, {10, 10}}, rotation = 0)));
+  Modelica.Blocks.Interfaces.BooleanInput frozen(start = false) "True if the integration is blocked" annotation(
+    Placement(visible = true, transformation(origin = {-110, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0), iconTransformation(origin = {-104, 0}, extent = {{-10, -10}, {10, 10}}, rotation = 0)));
   Interfaces.RealOutput y(start = Y0) "Output of the PI controller." annotation(
```

Comment:
You can write Modelica.Blocks.Interfaces.RealOutput and remove the related import.